dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -49,12 +50,15 @@
 import io.swagger.client.model.Label;
 import io.swagger.client.model.SourceFile;
 import io.swagger.client.model.ToolDescriptor;
+import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.FilenameUtils;
 import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.json.JSONObject;
```

Comment:
Looks like we already import GSON (and use it in a bunch of places), 
I think its a simple replacement for the way this uses JSONObject

Reply:
This is from moving a method that used JSONObject from another WorkflowClient into AbstractEntry. WorkflowClient used both json and gson. I'm not sure the reason though.